bio/varscan/somatic/wrapper.py

```
@@ -6,45 +6,30 @@
-in_pileup = ""
-mpileup = ""
-if "mpileup" in snakemake.input.keys():
-    # Case there is a mpileup with both normal and tumor
-    in_pileup = snakemake.input.mpileup
-    mpileup = "--mpileup 1"
-elif all(pileup in snakemake.input.keys() for pileup in pileup_pair):
-    # Case there are two separate pileup files
-    in_pileup = " {snakemake.input.normal_pileup}" " {snakemakeinput.tumor_pileup} "
-else:
-    raise KeyError("Could not find either a mpileup, or a pair of pileup files")
+prefix = op.path.splitext(snakemake.output.snp)[0]
```

Ruff (0.8.2)

Undefined name `op`

(F821)